packages/edit-site/src/components/add-new-template/new-template-part.js

```diff
@@ -18,31 +21,54 @@ import CreateTemplatePartModal from '../create-template-part-modal';
 
 export default function NewTemplatePart( { postType } ) {
 	const [ isModalOpen, setIsModalOpen ] = useState( false );
+	const { createSuccessNotice, createErrorNotice } = useDispatch(
+		noticesStore
+	);
 
 	async function createTemplatePart( { title, area } ) {
 		if ( ! title ) {
+			createErrorNotice( __( 'Title is not defined.' ), {
```

Comment:
How would this happen? 🙂 I guess it's okay to leave it just in case.

Reply:
Yeah, it will never happen for regular users. Maybe we should just remove this check altogether.